docs/pages/application-access/controls.mdx

```diff
@@ -79,6 +79,6 @@ allow:
   - [OIDC](../enterprise/sso/oidc.mdx)
   - [ADFS](../enterprise/sso/adfs.mdx)
   - [Azure AD](../enterprise/sso/azuread.mdx)
-  - [GSuite](../enterprise/sso/azuread.mdx)
+  - [GSuite](../enterprise/sso/google-workspace.mdx)
```

Comment:
GSuite -> Google Workspace

Reply:
Then please also update:

* https://github.com/gravitational/teleport/pull/9831
* https://github.com/gravitational/teleport/pull/9829
* https://github.com/gravitational/teleport/pull/9828

Reply:
Fix this for all versions, probably they need to be review as well. Thank you.